classes/hunter.lua

```diff
@@ -3,7 +3,7 @@ local AddonName, SAO = ...
 local function registerClass(self)
     local aimedShot = 19434;
     local arcaneShot = 3044;
-    local killShot  = 61006;
+    local kill_shot  = 61006;

```

Comment:
`kill_shot` only for the string passed to RegisterAura and RegisterCounter. For variable name, camelCase still applies i.e., `killShot`. Sorry if I wasn't clear.

Reply:
Done